[2025-01-16T20:15:38.977+0000] {processor.py:186} INFO - Started process (PID=9673) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:15:38.978+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:15:38.980+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:15:38.980+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:15:39.046+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:15:39.032+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:15:39.060+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:15:39.078+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.107 seconds
[2025-01-16T20:16:09.516+0000] {processor.py:186} INFO - Started process (PID=9740) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:16:09.517+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:16:09.519+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:16:09.518+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:16:09.543+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:16:09.535+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:16:09.547+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:16:09.563+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.053 seconds
[2025-01-16T20:16:39.963+0000] {processor.py:186} INFO - Started process (PID=9807) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:16:39.964+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:16:39.965+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:16:39.965+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:16:39.989+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:16:39.980+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:16:39.991+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:16:40.007+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.049 seconds
[2025-01-16T20:17:10.381+0000] {processor.py:186} INFO - Started process (PID=9874) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:17:10.382+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:17:10.384+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:17:10.383+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:17:10.412+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:17:10.401+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:17:10.415+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:17:10.434+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.059 seconds
[2025-01-16T20:17:41.139+0000] {processor.py:186} INFO - Started process (PID=9941) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:17:41.140+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:17:41.142+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:17:41.141+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:17:41.168+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:17:41.158+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:17:41.170+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:17:41.197+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.063 seconds
[2025-01-16T20:18:11.334+0000] {processor.py:186} INFO - Started process (PID=10008) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:18:11.336+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:18:11.337+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:18:11.337+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:18:11.374+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:18:11.359+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:18:11.378+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:18:11.399+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.072 seconds
[2025-01-16T20:18:41.529+0000] {processor.py:186} INFO - Started process (PID=10070) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:18:41.530+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:18:41.532+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:18:41.532+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:18:41.575+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:18:41.559+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:18:41.578+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:18:41.599+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.078 seconds
[2025-01-16T20:19:11.726+0000] {processor.py:186} INFO - Started process (PID=10136) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:19:11.727+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:19:11.729+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:19:11.728+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:19:11.768+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:19:11.752+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:19:11.772+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:19:11.801+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.083 seconds
[2025-01-16T20:19:41.896+0000] {processor.py:186} INFO - Started process (PID=10199) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:19:41.897+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:19:41.899+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:19:41.899+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:19:41.924+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:19:41.915+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:19:41.927+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:19:41.944+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.054 seconds
[2025-01-16T20:20:12.124+0000] {processor.py:186} INFO - Started process (PID=10266) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:20:12.126+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:20:12.129+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:20:12.128+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:20:12.230+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:20:12.182+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:20:12.236+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:20:12.285+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.172 seconds
[2025-01-16T20:20:42.776+0000] {processor.py:186} INFO - Started process (PID=10333) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:20:42.777+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:20:42.779+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:20:42.779+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:20:42.803+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:20:42.795+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:20:42.805+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:20:42.834+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.064 seconds
[2025-01-16T20:21:13.311+0000] {processor.py:186} INFO - Started process (PID=10399) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:21:13.314+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:21:13.316+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:21:13.316+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:21:13.342+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:21:13.333+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:21:13.346+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:21:13.372+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.066 seconds
[2025-01-16T20:21:44.071+0000] {processor.py:186} INFO - Started process (PID=10456) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:21:44.073+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:21:44.074+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:21:44.074+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:21:44.104+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:21:44.093+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:21:44.107+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:21:44.137+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.073 seconds
[2025-01-16T20:22:15.108+0000] {processor.py:186} INFO - Started process (PID=10519) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:22:15.110+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:22:15.111+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:22:15.111+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:22:15.137+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:22:15.126+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:22:15.141+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:22:15.158+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.056 seconds
[2025-01-16T20:22:46.107+0000] {processor.py:186} INFO - Started process (PID=10584) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:22:46.107+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:22:46.110+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:22:46.109+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:22:46.135+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:22:46.124+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:22:46.137+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:22:46.164+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.063 seconds
[2025-01-16T20:23:16.495+0000] {processor.py:186} INFO - Started process (PID=10645) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:23:16.496+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:23:16.498+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:23:16.497+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:23:16.525+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:23:16.514+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:23:16.527+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:23:16.555+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.065 seconds
[2025-01-16T20:23:46.782+0000] {processor.py:186} INFO - Started process (PID=10712) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:23:46.783+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:23:46.784+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:23:46.784+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:23:46.812+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:23:46.800+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:23:46.815+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:23:46.837+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.060 seconds
[2025-01-16T20:24:16.940+0000] {processor.py:186} INFO - Started process (PID=10779) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:24:16.941+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:24:16.942+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:24:16.942+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:24:16.966+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:24:16.957+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:24:16.968+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:24:16.985+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.050 seconds
[2025-01-16T20:24:47.330+0000] {processor.py:186} INFO - Started process (PID=10846) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:24:47.332+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:24:47.334+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:24:47.333+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:24:47.368+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:24:47.356+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:24:47.370+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:24:47.389+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.066 seconds
[2025-01-16T20:25:17.646+0000] {processor.py:186} INFO - Started process (PID=10913) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:25:17.647+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:25:17.648+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:25:17.648+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:25:17.675+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:25:17.663+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:25:17.677+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:25:17.708+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.067 seconds
[2025-01-16T20:25:47.892+0000] {processor.py:186} INFO - Started process (PID=10980) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:25:47.893+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:25:47.894+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:25:47.894+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:25:47.928+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:25:47.917+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:25:47.930+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:25:47.953+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.067 seconds
[2025-01-16T20:26:18.788+0000] {processor.py:186} INFO - Started process (PID=11047) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:26:18.789+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:26:18.790+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:26:18.790+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:26:18.826+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:26:18.816+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:26:18.831+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:26:18.861+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.078 seconds
[2025-01-16T20:26:49.135+0000] {processor.py:186} INFO - Started process (PID=11114) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:26:49.136+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:26:49.137+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:26:49.137+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:26:49.161+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:26:49.152+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:26:49.164+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:26:49.191+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.062 seconds
[2025-01-16T20:27:20.094+0000] {processor.py:186} INFO - Started process (PID=11182) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:27:20.095+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:27:20.097+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:27:20.096+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:27:20.128+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:27:20.116+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:27:20.130+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:27:20.161+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.072 seconds
[2025-01-16T20:27:50.757+0000] {processor.py:186} INFO - Started process (PID=11249) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:27:50.758+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:27:50.760+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:27:50.759+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:27:50.793+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:27:50.783+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:27:50.795+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:27:50.824+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.071 seconds
[2025-01-16T20:28:21.002+0000] {processor.py:186} INFO - Started process (PID=11316) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:28:21.003+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:28:21.005+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:28:21.004+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:28:21.034+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:28:21.022+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:28:21.037+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:28:21.055+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.060 seconds
[2025-01-16T20:28:51.302+0000] {processor.py:186} INFO - Started process (PID=11383) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:28:51.303+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:28:51.305+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:28:51.304+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:28:51.328+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:28:51.319+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:28:51.331+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:28:51.357+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.060 seconds
[2025-01-16T20:29:22.209+0000] {processor.py:186} INFO - Started process (PID=11451) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:29:22.210+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:29:22.212+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:29:22.211+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:29:22.242+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:29:22.230+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:29:22.245+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:29:22.273+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.071 seconds
[2025-01-16T20:29:52.979+0000] {processor.py:186} INFO - Started process (PID=11518) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:29:52.980+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:29:52.982+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:29:52.982+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:29:53.014+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:29:53.001+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:29:53.017+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:29:53.046+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.074 seconds
[2025-01-16T20:30:23.456+0000] {processor.py:186} INFO - Started process (PID=11585) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:30:23.458+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:30:23.459+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:30:23.458+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:30:23.488+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:30:23.476+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:30:23.490+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:30:23.518+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.067 seconds
[2025-01-16T20:30:53.712+0000] {processor.py:186} INFO - Started process (PID=11652) to work on /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:30:53.713+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/Transform.py for tasks to queue
[2025-01-16T20:30:53.714+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:30:53.714+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:30:53.741+0000] {logging_mixin.py:190} INFO - [2025-01-16T20:30:53.731+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/Transform.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/Transform.py", line 4084, in <module>
    "execution_count": null,
                       ^^^^
NameError: name 'null' is not defined
[2025-01-16T20:30:53.743+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/Transform.py
[2025-01-16T20:30:53.761+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/Transform.py took 0.054 seconds
